internal/website/content/internal/cmd/gocdk/static/demo/blob.bucket/_index.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
I believe once you've addressed the directory naming issue you can remove this file. Otherwise, we should amend the presubmit check.

Reply:
Done.